### PR TITLE
FuseML chart: versioning + logo

### PR DESCRIPTION
**Pull request by jsuchome**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -17,7 +17,8 @@ jobs:
 
       - name: Install FuseML Chart
         run: |
-          helm install fuseml fuseml/ --create-namespace --namespace fuseml-installer --set extensions.mlflow=true --set extensions.kfserving=true
+          latest=$(find -name Chart.yaml | cut -f 3 -d "/" | sort | tail -n 1)
```
[the remark this answers is not in the document]
Comment (flaviodsr) [reply .github/workflows/ci.yml:20]: So you are saying that can be many versions of the chart under `fuseml/` ?

[the remark this answers is not in the document]
Comment (jsuchome) [reply .github/workflows/ci.yml:20]: Hm, true. I guess I'd need to do some git magic then...

[the remark this answers is not in the document]
Comment (jsuchome) [reply .github/workflows/ci.yml]: Check now

[the remark this answers is not in the document]
Comment (flaviodsr) [reply .github/workflows/ci.yml]: ok, I think that is good enought, but there is also the case where two versions are modified.

**Pull request merged**